runtime-modules/content/src/lib.rs

```diff
@@ -1321,6 +1324,8 @@ decl_module! {
             video_id: T::VideoId,
             params: NftIssuanceParameters<T>
         ) {
+            // TODO AUDIT3: remove after open auction fix
+            unimplemented!();
```

Comment:
Please, change to the `DispatchError::Other`

Reply:
fixed in 93de823c63787575d87f47916ccd33f7530469be